build-tools/automation/azure-pipelines-oss.yaml

```diff
@@ -29,6 +29,8 @@ variables:
   DotNetCoreVersion: 3.1.201
   # Version number from: https://github.com/dotnet/installer#installers-and-binaries
   DotNetCorePreviewVersion: 5.0.100-rc.2.20459.1
+  GitHub.Token: $(github--pat--vs-mobiletools-engineering-service2)
```

Comment:
@pjcollins Alternatively, this could be a variable added to the build definition (same goes for the change to azure-pipelines.yaml). Let me know what you prefer.

Reply:
I think this works fine as is, I generally prefer these types of variables to be defined in the build definition UI only if they have different values across different builds.